docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1947,12 +1947,14 @@ When `-ingester.error-sample-rate` is configured to a value greater than `0`, th
 This error occurs when execution of a query exceeds the limit on the number of series chunks fetched.
 
 This limit is used to protect the system’s stability from potential abuse or mistakes, when running a query fetching a huge amount of data.
-To configure the limit on a per-tenant basis, use the `-querier.max-fetched-chunks-per-query` option (or `max_fetched_chunks_per_query` in the runtime configuration).
+To configure the limit on a global basis, use the `-querier.max-fetched-chunks-per-query` option.
+To configure the limit on a per-tenant basis, set `max_fetched_chunks_per_query` in the runtime configuration.
```

Comment:
Shall we mention `override`, since it's under `overrides` section of runtime config? (Same comment applies to all edits in the PR)

```suggestion
To configure the limit on a per-tenant basis, set `max_fetched_chunks_per_query` override in the runtime configuration.
```

Or maybe:
```suggestion
To configure the limit on a per-tenant basis, set `max_fetched_chunks_per_query` per-tenant override in the runtime configuration.
```

Reply:
Good idea, addressed in 56fdb1f.